src/it/resources/com/google/checkstyle/test/chapter5naming/rule522typenames/InputClassNames.java

```diff
@@ -0,0 +1,83 @@
+package com.google.checkstyle.test.chapter5naming.rule522typenames;
+
+class inputHeaderClass {
+    // violation above 'Type name 'inputHeaderClass' must match pattern '.*[A-Z][a-zA-Z0-9].*'.'
```

Comment:
Original pattern to match was `^[A-Z][a-zA-Z0-9]*$`

But it was causing error while verifying the message so I had to modify a little bit.

Reply:
please remvove '.*[A-Z][a-zA-Z0-9].*' at all no need to keep it, 
just `// violation above 'Type name 'inputHeaderClass' must match pattern`